LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2020-2021 PostHog Inc.
+Copyright (c) 2022-present PostHog Inc.
```

Comment:
Confirmed internally that this has to be specific years so `2020-2022` for now. Feel free update this if you want the PR credit otherwise I'll create a PR to make this change 

```suggestion
Copyright (c) 2020-2022 PostHog Inc.
```